### PR TITLE
colored_blocks_solution

### PR DESCRIPTION
**Pull request by eugeneSahach**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+}
+
+#red {
+  background-color: red;
+  height: 100px;
```
Comment (gestap) [inline src/style.css]: you duplicated this rule.
Create a common class for common style.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+#green {
```
Comment (gestap) [inline src/style.css]: for the future, do not name class `red, blue ...` or something like that.

**Comment by yevhen-sahach** [comment]
I've changed all thing that you suggested, please check it out.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+[class*="block"] {
```
Comment (gestap) [inline src/style.css]: better to create class instead of this structure `[class*="block"]`.
It would be easier to read and debug it

**Comment by yevhen-sahach** [comment]
done

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**